roles/maintenance_33_varnish/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignoreme
+    name: bar
+
+- <<: *task
+  vars:
+    taskid: 33-011
+    name: "Check which varnish configuration is active"
+  ansible.builtin.shell:
+    cmd: "systemctl show -p ExecStart varnish | grep 'ExecStart' | grep -oE '\\-f\\s*[a-z\\/\\.A-Z0-9]+' | grep -oE '[a-z\\/\\.A-Z0-9]{2,}'"
```

Comment:
Isn't the `grep 'ExecStart'` unneccessary here? `systemctl show -p ExecStart` should only return the ExecStart line to begin with.